samples/package.json

```diff
@@ -13,7 +13,9 @@
     "test": "mocha --timeout 600000 test/*.js"
   },
   "dependencies": {
-    "@google-cloud/aiplatform": "^1.0.0"
+    "@google-cloud/aiplatform": "^1.0.0",
+    "chai": "^4.2.0",
```

Comment:
I would expect `chai` to be a dev dependency

Reply:
Done.